src/librbd/api/Group.cc

```diff
@@ -39,9 +39,11 @@ template <typename I>
 snap_t get_group_snap_id(I* ictx,
                          const cls::rbd::SnapshotNamespace& in_snap_namespace) {
   ceph_assert(ceph_mutex_is_locked(ictx->image_lock));
-  auto it = ictx->snap_ids.lower_bound({in_snap_namespace, ""});
-  if (it != ictx->snap_ids.end() && it->first.first == in_snap_namespace) {
-    return it->second;
+  auto it = ictx->snap_ids.lower_bound({cls::rbd::SnapshotNamespace{}, ""});
```

Comment:
@dillaman I am not quite sure I understand using `lower_bound()` here. Won't it always return `begin()`?

Reply:
It skips past any `UserSnapshotNamespace` snapshots just due to the enum ordering.

Reply:
Oops -- typo